src/pages/JobCreate/JobCreateSystemsTable.tsx

```diff
@@ -26,27 +26,20 @@ const JobCreateSystemsTable = ({
         setSystems(response.data)
       })
       .catch((e) => {
-        setAlert({
-          severity: 'error',
-          message: e,
-          doNotAutoDismiss: true,
-        })
+        errorSetter(e)
       })
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
   return (
-    <>
-      <Table
-        tableKey="JobSystems"
-        data={useSystemsData(systems, systemSetter)}
-        columns={useSystemColumns()}
-        showGlobalFilter
-      >
-        {children}
-      </Table>
-      {alert && <Snackbar status={alert} />}
```

Comment:
I'm not sure if replacing the snack bar with an alert is appropriate here (and probably in several other places too). But I'm going to let it pass for the time being because there's another PR that's going to change the hooks substantially and I don't want you to waste time on something that will likely change.

Reply:
This is where it does the get to retrieve the system data and I was trying to add the alert where there was an error getting the initial data for a page.

Reply:
If this is the initial API call before the page loads, then I misspoke (mistyped?). In any case, we'll examine them all once this and the other PR is merged to make sure they make sense to us.